birdhouse/optional-components/README.rst

```diff
@@ -443,3 +443,54 @@ How to enable X-Robots-Tag Header in ``env.local`` (a copy from `env.local.examp
 
     .. seealso::
         See the `env.local.example`_ file for more details about this ``BIRDHOUSE_PROXY_ROOT_LOCATION`` behaviour.
+
+.. _prometheus-longterm-metrics
+
+Prometheus Long-term Metrics
+----------------------------
+
+This is a second prometheus instance that collects longterm monitoring metrics from the original prometheus instance
+(the one created by the ``components/monitoring`` component).
+
+Longterm metrics are any prometheus rule that have the label ``group: longterm-metrics`` or in other words are
+selectable using prometheus' ``'{group="longterm-metrics"}'`` query filter. To add some default longterm metrics rules
+also enable the ``prometheus-longterm-rules`` component.
+
+You may also choose to create your own set of rules instead of, or as well as, the default ones. See how to 
+:ref:`add additional rules here <monitoring-customize-the-component>`.
+
+To configure this component:
+
+    * update the ``PROMETHEUS_LONGTERM_RETENTION_TIME`` variable to set how long the data will be kept by prometheus
+
+Enabling this component creates the additional endpoint ``/prometheus-longterm-metrics``.
+
+.. _prometheus-longterm-rules
+
+Prometheus Long-term Rules
+--------------------------
+
+This adds some default longterm metrics rules to the `prometheus` component for use by the `prometheus-longterm-metrics`
+component. These rules all have the label ``group: longterm-metrics``.
+
+To see which rules are added, check out the
+`optional-components/prometheus-longterm-rules/config/monitoring/prometheus.rules` file.
```

Comment:
Re-reading this following other comments in the PR, I am left only more confused...

Putting ourselves in the shoes of someone having *absolutely no idea about the implementation details* of each component and how they interact with each other, what must the user do to achieve some results (aka: the "To enable this optional-component:" of each other section)?

ie: 
- If user wants SHORT only, what must be enabled (guessing only `components/monitoring`?)
- If user wants SHORT + LONG all locally, what must be enabled (`components/monitoring` + `optional-components/prometheus-longterm-metrics` ?) (+ `optional-components/prometheus-longterm-rules` depending on other dependency comment)
- If user wants SHORT local and LONG remote (`components/monitoring` + `optional-components/prometheus-longterm-rules` + IMPORTANT `PROMETHEUS_LONGTERM_TARGETS` override ?) 

--- 

Specifically regarding the "Prometheus Long-term Rules" section. 
It is mentioned that they are applied to `prometheus`. (this is fine, I understand why to apply the group tags)

However, *for a user without the details of "why"*, it is very confusing! I can see a user think they "should logically" be applied to `prometheus-longterm-rules` service given their names. A much better explanation must be given. 

Basically, something must indicate that service `prometheus` is the source that feeds the others instances using `optional-components/prometheus-longterm-rules`. Then, using  `optional-components/prometheus-longterm-metrics`, they can decide that this receiving instance is another local one (ie: `prometheus-longterm-metrics` service), or dispatch it remotely by overriding `PROMETHEUS_LONGTERM_TARGETS` accoridngly.

Reply:
Ok... just to clarify what you're asking for here ...

You would like specific instructions about how to set up each of these three cases:

> - If user wants SHORT only ...
> - If user wants SHORT + LONG all locally ...
> - If user wants SHORT local and LONG remote ...

And you would like an additional description of how the various monitoring components interact?
Is there anything else?

Reply:
That's it. Just mitigate user expectations and ensure there is a common understanding of the components/services.